cannot find package "github.com/google/git-appraise-web/api" when running make

```~/OpenSource $ git clone https://github.com/google/git-appraise-web.git
Cloning into 'git-appraise-web'...
remote: Counting objects: 445, done.
remote: Total 445 (delta 0), reused 0 (delta 0), pack-reused 445
Receiving objects: 100% (445/445), 378.44 KiB | 149.00 KiB/s, done.
Resolving deltas: 100% (250/250), done.
Checking connectivity... done.
~/OpenSource $ cd git-appraise-web/
~/OpenSource/git-appraise-web@master $ make
go get -u github.com/jteeuwen/go-bindata/...
go get -u github.com/google/git-appraise/...
rm assets/*~ 2>/dev/null || true
/Users/pablo/go/bin/go-bindata -modtime 1 -pkg assets -o third_party/assets/assets.go assets/
gofmt -w ./
go vet ./...
go test ./...
git-appraise-web/git-appraise-web.go:28:2: cannot find package "github.com/google/git-appraise-web/api" in any of:
        /usr/local/opt/go/libexec/src/github.com/google/git-appraise-web/api (from $GOROOT)
        /Users/pablo/go/src/github.com/google/git-appraise-web/api (from $GOPATH)
git-appraise-web/git-appraise-web.go:29:2: cannot find package "github.com/google/git-appraise-web/third_party/assets" in any of:
        /usr/local/opt/go/libexec/src/github.com/google/git-appraise-web/third_party/assets (from $GOROOT)
        /Users/pablo/go/src/github.com/google/git-appraise-web/third_party/assets (from $GOPATH)
make: *** [test] Error 1```

--- a/README.md
+++ b/README.md
@@ -16,15 +16,26 @@ Building requires the Go tools and GNU Make. Running the built binary requires t
 
 ## Building the source code
 
-First checkout the code from the git repo:
+Assuming you have the [Go tools installed](https://golang.org/doc/install), run
+the following command:
 
-    git clone git@github.com:google/git-appraise-web.git
+    go get github.com/google/git-appraise-web/git-appraise-web
+
+### Manual steps
+
+Assuming you have not run the above command, first checkout the code from the git repo:
+
+    mkdir -p ${GOPATH}/src/github.com/google
+    git clone https://github.com/google/git-appraise-web.git
 
 Build the binary:
 
+    cd ${GOPATH}/src/github.com/google/git-appraise-web
     make
 
-And then launch it:
+## Running the application
+
+Binary is placed into `${GOPATH}/bin`:
 
     ${GOPATH}/bin/git-appraise-web
 

